const ErrorHandler = require("../../utils/errorHandler")
const catchAsyncError = require("../../utils/catchAsyncError")
const invoiceModel = require("./invoice.model");

exports.create = catchAsyncError(async(req, res, next) => {

})

exports.index = catchAsyncError(async(req, res, next) => {
  
})

exports.show = catchAsyncError(async(req, res, next) => {
  
})

exports.update = catchAsyncError(async(req, res, next) => {
  
})

exports.destroy = catchAsyncError(async(req, res, next) => {
  
})